cmd/pdatagen/internal/base_fields.go

```diff
@@ -123,21 +123,21 @@ type baseField interface {
 	generateCopyToValue(sb *strings.Builder)
 }
 
-type sliceField struct {
+type slicePtrField struct {
```

Comment:
What do you think about naming this:
```suggestion
type sliceOfPtrsField struct {
```

Reply:
Done.

Reply:
Looks like it is still called `slicePtrField`. Not a strong opinion but I think `sliceOfPtrsField` reflects the intent a bit better. Up to you.

Reply:
Oh.. changed the slice structs not the fields.

Reply:
Updated, the field is actually for both slices.